docs/questions/alerts.md

```diff
@@ -57,8 +66,6 @@ You can choose:
 - Whether you want Metabase to alert you every time the time series crosses a goal line, or only the first time it crosses the goal line.
 - How often you want Metabase to check to see if the goal line has been crossed.
 
-![Goal line alert options](./images/goal-line-options.png)
-
 Click Done, and your alert will be all set up! You'll get an email confirmation, too. If you need to edit or unsubscribe from the alert you set up, just click that same icon. You'll see the Edit and Unsubscribe buttons. This is also where you'll see alerts about this question that administrators might have added you to.
```

Comment:
"you'll get an email confirmation" reminded me: should we talk about all the different emails you get regarding alerts?
there are
- You set up an alert
- You’ve been unsubscribed from an alert
- One of your alerts has stopped working
- You unsubscribed from an alert
- Admin added you to an alert
?